unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -500,6 +500,36 @@ public static IntPtr unity_method_get_function_pointer([NativeCallbackType("Mono
         return IntPtr.Zero;
     }
 
+    [NativeFunction("coreclr_method_full_name")]
+    public static void coreclr_method_full_name(
+        [NativeCallbackType("MonoMethod*")] IntPtr method,
+        bool signature,
+        [NativeCallbackType("void*")] void* buffer,
+        [NativeCallbackType("AssignString")] delegate* unmanaged[Cdecl]<void*, char*, int, void> assignString )
+    {
+        MethodBase mb = MethodBase.GetMethodFromHandle(method.MethodHandleFromHandleIntPtr());
+        StringBuilder retVal = null;
+        if (mb != null)
+        {
+            retVal = new StringBuilder($"{mb.ReflectedType?.Namespace}.{mb.ReflectedType?.Name}:{mb.Name}");
```

Comment:
I think this is wrong and will always include a `.` before the type name, event when a namespace is not present.